tests/store/tracking/test_sqlalchemy_store.py

```diff
@@ -143,6 +145,139 @@ def test_fail_on_multiple_drivers():
         extract_db_type_from_uri("mysql+pymsql+pyodbc://...")
 
 
+@dataclass
+class TestDatabase:
+    db_url: str
+    store: SqlAlchemyStore
+
+
+@pytest.fixture
+def test_database() -> TestDatabase:
+    temp_dbfile, db_url = _setup_db_uri()
+    store = _get_store(db_url)
+    yield TestDatabase(db_url, store)
+    _cleanup_database(temp_dbfile, store)
+
+
+def _get_store(db_uri=""):
```

Comment:
We may be able to reuse the function of `TestSqlAlchemyStore` to make the temporal code DRY, but I didn't try it since the class itself will be removed soon.

Reply:
SGTM